### ISSUE TITLE
Unable to fetch tags data. 

### ISSUE THREAD

**Issue by wrfly** [SUSE/Portus#1532]
Portus can not fetch image tags, 500 error.

Portus Version: `2.3.0-dev`

error log:
```
Attaching to portus_portus_1
portus_1    | Started GET "/repositories/76.json" for 60.247.122.90 at 2017-12-11 10:11:25 +0000
portus_1    | Processing by RepositoriesController#show as JSON
portus_1    |   Parameters: {"id"=>"76"}
portus_1    | Started GET "/v2/token?account=portus&scope=repository%3Aplatform%2Fdeployment%3Apull&service=docker.company.net" for 172.18.0.8 at 2017-12-11 10:11:25 +0000
portus_1    | Processing by Api::V2::TokensController#show as JSON
portus_1    |   Parameters: {"account"=>"portus", "scope"=>"repository:platform/deployment:pull", "service"=>"docker.company.net"}
portus_1    | Completed 200 OK in 78ms (Views: 0.2ms | ActiveRecord: 2.8ms)
portus_1    | Completed 500 Internal Server Error in 170ms (ActiveRecord: 1.4ms)
portus_1    |
portus_1    | NoMethodError (undefined method `each' for nil:NilClass):
portus_1    |   lib/portus/security_backends/clair.rb:43:in `layer_vulnerabilities'
portus_1    |   lib/portus/security_backends/clair.rb:30:in `vulnerabilities'
portus_1    |   lib/portus/security.rb:48:in `block in vulnerabilities'
portus_1    |   lib/portus/security.rb:46:in `each'
portus_1    |   lib/portus/security.rb:46:in `vulnerabilities'
portus_1    |   app/models/tag.rb:96:in `vulnerabilities'
portus_1    |   app/models/tag.rb:90:in `block in as_json'
portus_1    |   app/models/tag.rb:89:in `tap'
portus_1    |   app/models/tag.rb:89:in `as_json'
portus_1    |   app/controllers/repositories_controller.rb:19:in `show'
```

**Comment by mssola** [comment]
This issue has been already (theoretically 😉) fixed. In `master` at least this should not happen (now, bear in mind that `opensuse/portus:head` is a bit behind because of #1418).

So, is this from the `docker-compose.yml` file we provide at the root of the project (development version), or this comes from a `docker-compose.yml` file taking `opensuse/portus:head` ?

Finally, since you are using Clair, this is not documented properly (see #1534 and #1535), but you should be using v2.0.x 😅.

**Comment by wrfly** [comment]
Hi, this issue came with `Clair`, when I remove `Clair`, everything is  #fine.
The `opensuse/portus`'s digest is `sha256:c144b4ea871e3ddbeb8b9f4acaa002cb...`

**Comment by mssola** [comment]
Yeah, I just looked into it and it's because `opensuse/portus:head` is actually behind master: master should work.

Hence, I'll add this issue as a duplicate of #1418, so when we fix that (soon enough) we can close this one too. Thanks for reporting.

If you want to get your hands dirty meanwhile, maybe you could try building the docker image from the Dockerfile we provide on the root of the project. That should work fine (even if it's a bit larger in size). You can even use the docker-compose.yml files we provide (the main development one is also using clair).

**Comment by wrfly** [comment]
Thank you very much!

**Comment by mssola** [comment]
Fixed! See my comments on #1418
